metricbeat/module/system/raid/raid.go

```diff
@@ -67,33 +69,61 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 		return nil, err
 	}
 
+	sysMountPoint := filepath.Join(config.MountPoint, "/sys/block")
+
 	return &MetricSet{
 		BaseMetricSet: base,
 		fs:            fs,
+		sysblock:      sysMountPoint,
 	}, nil
 }
 
+//Annoyingly, different linux subsystems report size size using different blocks
+// /proc/mdstat /proc/partitions and mdadm (Via the BLKGETSIZE64 ioctl call) count "size" as the number of 1024-byte blocks.
+// /sys/block/md*/size uses 512-byte blocks. As does /sys/block/md*/md/sync_completed
+//convert the 512-byte blocks to 1024 byte blocks to maintain how this metricset "used to" report size
```

Comment:
These comments should start with the thing commented.

Reply:
I'm not sure what you mean?

Reply:
If this is documenting the function it should start with the name of the function, I see you have moved it inside the function, then I guess it is fine.